Keda.CosmosDB.Scaler/src/Repository/ChangeFeedEstimatorFactory.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Keda.CosmosDB.Scaler.Services;
+using Microsoft.Azure.Cosmos;
+using System;
+using System.Collections.Concurrent;
+
+namespace Keda.CosmosDB.Scaler.Repository
+{
+    public class ChangeFeedEstimatorFactory
+    {
+        private ConcurrentDictionary<CosmosDBTrigger, ChangeFeedEstimator> _changeFeedBuilderMap;
+        private static ChangeFeedEstimatorFactory _instance = new ChangeFeedEstimatorFactory();
+

```

Comment:
@divyagandhisethi  - not sure if you noticed feedback provided here: https://github.com/divyagandhisethi/keda-external-scaler-azure-cosmos-db/pull/1 --> adding here explicitly --> This class is used just for maintaining dictionary of estimators --> does not follow factory design pattern

Reply:
I have explained it here already as part of your previous comment - https://github.com/kedacore/keda-external-scaler-azure-cosmos-db/pull/1#discussion_r629593909.

Reply:
Missed your earlier response. Closing this comment. We can discuss in the comment you referenced above